Automattic/wp-calypso
# Scheduled updates multisite: list view

Related to https://github.com/Automattic/wp-calypso/issues/89734

![CleanShot 2024-04-25 at 17 08 04@2x](https://github.com/Automattic/wp-calypso/assets/528287/ea8cde91-8eb3-42bb-98af-9937758ea93d)

## Proposed Changes

- Shows table with main row containing schedule + details rows per site
- Adapted the date/time helpers to fallback to user's locale in multisite context.
- Added `{ countInstalledPlugins }` function, that counts the plugins from `schedule.args` that are actually installed (closes https://github.com/Automattic/wp-calypso/issues/89841)
- Loading state

## Todo in follow up PR's
- Layout of the parent page (Title & Add schedule button) https://github.com/Automattic/wp-calypso/issues/89964 
- Search filter component: https://github.com/Automattic/wp-calypso/issues/89966
- Optimistic updates: https://github.com/Automattic/wp-calypso/issues/89967
- Permission checking: https://github.com/Automattic/wp-calypso/issues/89968
- Error handling: https://github.com/Automattic/wp-calypso/issues/89969
- Mobile view: https://github.com/Automattic/wp-calypso/issues/89970
- Empty view: https://github.com/Automattic/wp-calypso/issues/89971
- Back button: https://github.com/Automattic/wp-calypso/issues/89982

## Testing Instructions

1. Apply this PR
2. Navigate to http://calypso.localhost:3000/plugins/scheduled-updates or use the Calypso Live URL with flag `plugins/multisite-scheduled-updates`
3. Play around. Creating & deleting schedules should work, but it's still bumpy so look at your network requests panel to see any failed schedule creations.

- Make sure there are no regressions in single site.

## Pre-merge Checklist
